pacta/build_with_tag.sh

```diff
@@ -1,51 +1,80 @@
 #! /bin/bash
 
-tag="${1:-latest}"
-clones="${2:-PACTA_analysis create_interactive_report StressTestingModelDev pacta-data}"
+# Examples:
+# build_with_tag 0.0.0.999
+
+repos="PACTA_analysis create_interactive_report StressTestingModelDev pacta-data"
+user_results="user_results"
 url="git@github.com:2DegreesInvesting/"
 
-for repo in ${clones}
-do
-    remote="${url}${repo}.git"
-    git clone -b master ${remote} --depth 1
-    echo "--"
-done
+tag="$1"
+if [ -z "$tag" ]
+then
+    echo "Please give a tag."
+    exit 0
+fi
```

Comment:
I believe you suggested that it makes sense to always create a new tag, so I'm enforcing it.

Reply:
agreed